packages/components/src/components/form/PhoneInput/PhoneInput.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { PhoneInput } from './PhoneInput';
+import { IconPhone } from '../../media/Icons';
+
+export default {
+  component: PhoneInput,
+  title: 'Components/Form/PhoneInput',
+} as Meta;
+
+const Template = ({ showLeadingIcon, countryIsoCode, ...args }) => (
+  <PhoneInput
+    maskOptions={{
+      countryIsoCode: countryIsoCode || 'US',
+    }}
+    {...{
+      ...(showLeadingIcon && {
+        leadingIcon: <IconPhone />,
+      }),
+    }}
+    {...args}
+  />
+);
+
+export const Default = Template.bind({});
+
+export const Playground = Template.bind({});
+Playground.args = {
+  variant: 'outline',
+  size: 'l',
+  countryIsoCode: 'US',
```

Comment:
can we make this a select with some options?, the storybook crashes when you change this value